cp-quickstart/start-docker-kraft.sh

```diff
@@ -19,8 +19,10 @@ sleep 2 # give connect an exta moment to fully mature
 echo "connect has started!"
 
 # Configure datagen connectors
-source ./connectors/submit_datagen_pageviews_config.sh
-source ./connectors/submit_datagen_users_config.sh
+wget -O connector_pageviews_cos.config  https://github.com/confluentinc/kafka-connect-datagen/raw/master/config/connector_pageviews_cos.config
```

Comment:
Why not use the scripts here anymore? The kraft quickstart seems to work ok using the scripts, i.e., with this change:
```
-wget -O connector_pageviews_cos.config  https://github.com/confluentinc/kafka-connect-datagen/raw/master/config/connector_pageviews_cos.config
-curl -X POST -H "Content-Type: application/json" --data @connector_pageviews_cos.config http://localhost:8083/connectors
-wget -O connector_users_cos.config https://github.com/confluentinc/kafka-connect-datagen/raw/master/config/connector_users_cos.config
-curl -X POST -H "Content-Type: application/json" --data @connector_users_cos.config http://localhost:8083/connectors
+source ./connectors/submit_datagen_pageviews_config.sh
+source ./connectors/submit_datagen_users_config.sh
```

Reply:
Yeah, but that one has interceptors, and C3 wasn't part of the docker-compose.  But see other comment, do we want C3 in or out?  We can add it in if you think it's helpful

Reply:
chatted IRL about this, result being that you're going to add C3